### DISCUSSION TITLE
Foundry Error: Member "startBroadCast" not found or not visible after argument-dependent lookup in contract Vm.

### DISCUSSION THREAD

**Discussion by KingVics** [Cyfrin/foundry-full-course-cu#3011] [General]
Hi community, I am trying to compile SimpleStorage.sol foundry fundamental but running into an error with my forge-std lib. 
Tried remapping forge and other tweaks but to no avail. Running windows 11.
```
// SPDX-License-Identifier: MIT
pragma solidity >=0.6.2 <0.9.0;

import "../src/SimpleStorage.sol";
import 'forge-std/Script.sol';

contract DeployS is Script {
     function run(SimpleStorage) external {
        vm.startBroadCast();
        SimpleStorage simpleStorage = new SimpleStorage();
        vm.stopBroadCast();
        return simpleStorage;
    }

}
```
Source "src/SimpleStorage.sol" not found: File not found. Searched the following locations:
Source "lib/forge-std/src/Script.sol" not found: File not found. Searched the following locations: ""

**Comment by heheboii11** [comment]
Can you do named imports and please use ```""``` instead of ```''```,  also please add return to the ```run()``` function and define SimpleStorage contract in DeployS contract. So your Script should something look like below
```solidity
import {SimpleStorage} from "../src/SimpleStorage.sol";
import {Script} from "forge-std/Script.sol";

contract DeployS is Script {
SimpleStorage simplestorage
     function run() external returns (SimpleStorage){
        vm.startBroadCast();
        SimpleStorage simpleStorage = new SimpleStorage();
        vm.stopBroadCast();
        return simpleStorage;
    }

}

**Reply by KingVics** [reply]
Thanks, got it to work. Turns out my solidity version was the issue.

**Comment by EngrPips** [comment]
Hey @KingVics, does @heheboii11 suggestion  resolves your issue already?

**Reply by KingVics** [reply]
Issued resolved. Thanks

**Reply by EngrPips** [reply]
Awesome, I am glad you figured it out. Keep crushing it.
